src/go/bank.go

```diff
@@ -0,0 +1,1472 @@
+package ifsc
+
+import (
+	"fmt"
+	"log"
+)
+
+var bankCodes = map[string]int{
```

Comment:
Anyway to avoid duplicating the list here? Don't want to re-create this in so many places. Can we iterate over the constants to generate this instead?

Reply:
@captn3m0 If we iterate the constants in go we won't be able to use them as `Bank Constants`. But we can do that in a struct. So, I have created a struct which has the field names as the bank codes. It solves the problem.

Reply:
@captn3m0 what do you think?